Signal/src/AppDelegate.m

```diff
@@ -192,6 +192,24 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     return YES;
 }
 
+
+/**
+ *  The user must unlock the device once after reboot before the database encryption key can be accessed.
+ */
+- (void)verifyDBKeysAvailableBeforeBackgroundLaunch
+{
+    if (CurrentAppContext().isMainApp && CurrentAppContext().mainApplicationState != UIApplicationStateBackground) {
```

Comment:
nit: this CurrentAppContext is redundant again.

Reply:
Done.